.github/workflows/deploy-hs.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+name: Hosting service - Deploy
+jobs:
+  hostingServiceDeploy:
+    name: Deployment bot
+    environment: dev
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+      - name: Deployment bot
+        uses: ./.github/hs-deploy-action
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is one of the secrets value `spartacus`?

Because of that the logs are messed up:
![image](https://user-images.githubusercontent.com/6259079/109047570-510fc900-76d6-11eb-9b5f-8dc9bacfddda.png)

Every `spartacus` occurance got replaced with `***`

Reply:
Yes